src/main/java/mops/gruppen2/domain/model/group/wrapper/Description.java

```diff
@@ -0,0 +1,27 @@
+package mops.gruppen2.domain.model.group.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Value
+public class Description {
+
+    @NotNull
+    @Size(min = 4, max = 512)
+    @JsonProperty("value")
+    String value;
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @JsonIgnore
+    public static Description EMPTY() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'EMPTY' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/tobi_2/gruppenfindung/pullRequest?prid=5356872)